src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -627,7 +627,7 @@ bool BufferUntil(byte toFind, StdInReader src, ref Span<byte> dst, ref int dstPo
                         }
 
                         // Otherwise, push it back into the reader's extra buffer.
-                        reader.AppendExtraBuffer(&b, 1);
+                        reader.AppendExtraBuffer(MemoryMarshal.CreateReadOnlySpan(ref b, 1));
```

Comment:
Can 'unsafe' keyword be removed in any more places in this file? Eg., this method.

Reply:
Addressed.